src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -19,6 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" />
+    <PackageReference Include="System.Diagnostics.Debug" />
```

Comment:
May I know what's this for? Seems no code change in this project?

Reply:
Elevating our msbuild dependencies led to a downgrade because of this: https://github.com/NuGet/Home/issues/3541. 

The NU1605 guidance is to update to the higher version of the package to avoid the downgrade.